src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1641,7 +1641,7 @@ internal enum ErrorCode
         WRN_NullReferenceReturn = 8603,
         WRN_NullReferenceArgument = 8604,
         WRN_UnboxPossibleNull = 8605,
-        WRN_NullReferenceIterationVariable = 8606,
+        // available 8606,
```

Comment:
The `ErrorCode` probably shouldn't be available for re-use because there may be existing code with `#pragma warning disable 8606` that would disable a distinct warning in the future.

Reply:
Consider commenting out the obsolete `ErrorCode` as we do elsewhere:
`WRN_NullReferenceIterationVariable = 8606,`

---
In reply to: [354997117](https://github.com/dotnet/roslyn/pull/40207#discussion_r354997117) [](ancestors = 354997117)